Sources/SwiftProtobuf/Internal.swift

```diff
@@ -0,0 +1,42 @@
+// Sources/SwiftProtobuf/Message.swift - Message support
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See LICENSE.txt for license information:
+// https://github.com/apple/swift-protobuf/blob/master/LICENSE.txt
+//
+// -----------------------------------------------------------------------------
+///
+/// Internal helpers on Messages for the library. These are public
+/// just so the generated code can call them, but shouldn't be called
+/// by developers directly.
+///
+// -----------------------------------------------------------------------------
+
+
+import Foundation
+import Swift
+
+public struct Internal {
```

Comment:
@allevato @tbkka Does it make sense to move any other stuff here?  Especially some of the helper types used in parsing/generating?  I assume they don't have to physically move, just get updated to nest them in an extension of this.